src/minimum.jl

```diff
@@ -0,0 +1,127 @@
+"""
+```
+t = find_threshold(MinThreshold(),  histogram, edges)
+```
+
+Under the assumption that the histogram is bimodal the histogram is smoothed using 3-mean smoothing untill modes remain. The threshold is then set such that it is the minumum value between these two modes.

```

Comment:
Spelling mistake here as well. Perhaps use the update first sentence from `Intermodes`.